javascript/console_output/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "console_output",
+  "version": "1.0.0",
+  "description": "",
+  "main": "console_output.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
what does this `test`scripts do?

Reply:
this is auto created when doing npm init

Reply:
Hmm, OK...